hatchling-web-app/src/components/TextEditor.jsx

```diff
@@ -78,12 +103,17 @@ export default function Texteditor() {
               />
             )}
           </button>
-          <button className="bg-cyan-900 text-white border-2 border-cyan-900 rounded flex items-center gap-1 p-2 text-xs sm:text-sm md:text-base hover:scale-105 active:scale-95">
+
+          <input type='file' id='file' ref={inputFile} onChange={handelInputFile} style={{display: 'none'}}/>
+
+          <button onClick={onLoadButtonClick} className="bg-cyan-900 text-white border-2 border-cyan-900 rounded flex items-center gap-1 p-2 text-xs sm:text-sm md:text-base hover:scale-105 active:scale-95">
             Load{" "}
             <DescriptionIcon
               fontSize={windowSize.width >= MD_BREAKPOINT ? "large" : "small"}
             />
           </button>
+
+
```

Comment:
Please remove the unneeded whitespace, and run the command `npm run format` to fix the formating issues in this file.